helm/alfresco-process-infrastructure/README.md

```diff
@@ -360,4 +360,4 @@ Kubernetes: `>=1.15.0-0`
 | setup-acs-script-job.loadTestData | bool | `true` |  |
 
 ----------------------------------------------
-Autogenerated from chart metadata using [helm-docs v1.4.0](https://github.com/norwoodj/helm-docs/releases/v1.4.0)
+Autogenerated from chart metadata using [helm-docs v1.3.0](https://github.com/norwoodj/helm-docs/releases/v1.3.0)
```

Comment:
1 version lower?

Reply:
this is something was there but the doc was not updated.